src/upgrade/connection.rs

```diff
@@ -1,4 +1,4 @@
-use async_std::io::{self, prelude::*};
+use futures_lite::*;
```

Comment:
I don't love the wildcard imports for non-prelude crates; can we import the items individually instead?

Reply:
fixed